SPHINXsys/src/shared/particles/base_particles.h

```diff
@@ -82,7 +82,7 @@ namespace SPH
 	class BaseParticles
 	{
 	private:
-		UniquePtrKeepers<BaseDynamics<void>> derived_particle_data_; /**< Unique ptr for Base dynamics. */
+		UniquePtrsKeeper<BaseDynamics<void>> derived_particle_data_; /**< Unique ptr for Base dynamics. */
 		
 		template <typename VariableType>
 		struct AllVariablesDelete
```

Comment:
`AllVariablesDelete` should now be removed according to current changes

Reply:
Ok.